src/server/cli.ts

```diff
@@ -0,0 +1,166 @@
+      anchorPath,
+      initialLoadTimeout: serverSettings.pageMustLoadTimeout,
+      verbose,
+      logger: LOGGER
+    });
+    createServer(serverSettings, createApp(serverSettings, settingsManager, version), program);
+  });
+
+program
+  .command("run-examples")
+  .addOption(portOption)
+  .addOption(serverRootOption)
+  .addOption(serverHostOption)
+  .addOption(verboseOption)
+  .action(({ port, verbose, serverRoot, serverHost }) => {
+    const configPath = path.join(__dirname, "../../config-examples.yaml");
+    const anchorPath = path.dirname(configPath);
+    const config = loadConfigFile(configPath);
+    const { sources, serverSettings, appSettings } = buildSettings(config, { port, verbose, serverHost, serverRoot });
+    const settingsManager = new SettingsManager(appSettings, sources, {
+      anchorPath,
+      initialLoadTimeout: serverSettings.pageMustLoadTimeout,
+      verbose,
+      logger: LOGGER
+    });
+    createServer(serverSettings, createApp(serverSettings, settingsManager, version), program);
+  });
+
+program
+  .command("connect-druid")
+  .argument("<url>", "druid url")
+  .addOption(portOption)
+  .addOption(serverRootOption)
+  .addOption(serverHostOption)
+  .addOption(verboseOption)
+  .addOption(usernameOption)
+  .addOption(passwordOption)
+  .action((url, { port, verbose, username, password, serverRoot, serverHost }) => {
+    const auth = parseCredentials(username, password);
+    const { appSettings, serverSettings, sources } = settingsForDruidConnection(url, { port, verbose, serverHost, serverRoot }, auth);
+    const settingsManager = new SettingsManager(appSettings, sources, {
+      anchorPath: process.cwd(),
+      initialLoadTimeout: serverSettings.pageMustLoadTimeout,
+      verbose,
+      logger: LOGGER
+    });
+    createServer(serverSettings, createApp(serverSettings, settingsManager, version), program);
+  });
+
+program
+  .command("load-file")
+  .argument("<file>", "json file")
+  .requiredOption("-t, --time-attribute <attribute>", "time attribute")
+  .addOption(portOption)
+  .addOption(serverRootOption)
+  .addOption(serverHostOption)
+  .addOption(verboseOption)
+  .action((file, { timeAttribute, port, verbose, serverHost, serverRoot }) => {
+    const { appSettings, sources, serverSettings } = settingsForDatasetFile(file, timeAttribute, { serverRoot, serverHost, verbose, port });
+    const settingsManager = new SettingsManager(appSettings, sources, {
+      anchorPath: process.cwd(),
+      initialLoadTimeout: serverSettings.pageMustLoadTimeout,
+      verbose,
+      logger: LOGGER
+    });
+    createServer(serverSettings, createApp(serverSettings, settingsManager, version), program);
+  });
+
+program
+  .command("verify-config")
+  .argument("<file>", "path to config file")
+  .addOption(verboseOption)
+  .action((file, { verbose }) => {
+    try {
+      const config = loadConfigFile(file);
+      buildSettings(config, { verbose });
```

Comment:
For now, we either return from command with ok status code or we throw an error.